lib/site_prism/section.rb

```diff
@@ -11,9 +11,11 @@ class Section
 
     attr_reader :root_element, :parent
 
+    # rubocop:disable Naming/AccessorMethodName
```

Comment:
Lets remove this override and leave the rubocop rule as ignored in the suite (We may start introducing more)